federation-js/src/composition/compose.ts

```diff
@@ -58,6 +58,12 @@ const EmptyMutationDefinition = {
   fields: [],
   serviceName: null,
 };
+const EmptySubscriptionDefinition = {
+  kind: Kind.OBJECT_TYPE_DEFINITION,
+  name: { kind: Kind.NAME, value: 'Subscription' },
```

Comment:
Can we use a constant here?
Like this:

```
name: { kind: Kind.NAME, value: defaultRootOperationNameLookup.subscription },
```

Reply:
@Chase-C can I help you with this PR?

Reply:
Be my guest. I tried making the change you suggested, but I'm not sure it's possible to push new commits to this branch without creating an entirely new PR (this one was migrated from the apollo-server project).

Reply:
I tried here, and really need create a new PR with fork. Do you want to create there? Or do I create here?